src/reactpy/html.py

```diff
@@ -1,4 +1,7 @@
 """
+
+**Disclaimer** : Do not use raw user inputs from untrusted data sources in scripts. Doing so can lead to security vulnerabilities such as cross-site scripting (XSS) attacks.
```

Comment:
Can we check out how this renders with our auto docs before publishing?

I'm not sure how/if sphinx handles disclaimer tags.

Reply:
can you please link the file producing auto-docs?

Reply:
yeah! you were right! Making changes to it.

Reply:
Do check it now!